### ISSUE TITLE
Insufficient contrast in dart docs samples

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#26398]
The contrast between the light blue and white ([currently 2.86](https://contrast-ratio.com/#white-on-rgb%2860%2C%20160%2C%20228%29)) and the light blue and dark blue ([currently 1.83](https://contrast-ratio.com/#RGB%2831%2C%20105%2C%20208-on-rgb%2860%2C%20160%2C%20228%29)) in the sample boxes ([see example below](https://docs.flutter.io/flutter/widgets/NestedScrollView-class.html)) is too low for a11y. Should be at least 4.5.

<img width="731" alt="screen shot 2019-01-10 at 4 08 42 pm" src="https://user-images.githubusercontent.com/1227763/51005025-0a5cb600-14f2-11e9-89cd-ffa95131d901.png">
